src/source/tealiumCollect.brs

```diff
@@ -99,19 +100,14 @@ function CreateTealiumCollect(tealiumLog) as Object
                         m._Print(message, 3)
                     end if
                 end if
-                if callbackObj <> invalid then
-                    if callbackObj.DoesExist("callback") <> invalid then
-                        callbackObj.callback(event)
-                    end if
-                end if
-                return fullUrl
+                return url
```

Comment:
why did we remove the completion?

Reply:
The completion can't be used from another thread, so there would be no way for someone to pass a completion to our task that does the track calls since it takes it on another thread. That being said I may add it back for users that don't want to use our Task and have full control over the lib

Reply:
@trister1997 minor detail here, but why did we move the callback inside the `if urlTransfer.AsyncPostFromString`? Shouldn't the completion be called even if the method returns false?